staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -191,18 +198,18 @@ func (c *celAdmissionController) Validate(
 				message = fmt.Errorf("failed to configure binding: %w", err).Error()
 			}
 			deniedDecisions = append(deniedDecisions, policyDecisionWithMetadata{
-				PolicyDecision: PolicyDecision{
-					Action:  ActionDeny,
-					Message: message,
+				policyDecision: policyDecision{
```

Comment:
Why did these get re-cast to private? They were made public by a previous PR so that callers outside of validatingadmissionpolicy could process the results.

Reply:
It wasn't currently used anywhere so I thought I'd flip them but I can flip them back quick, done